blueprintapi/settings.py

```diff
@@ -122,6 +121,7 @@
 }
 
 AUTH_USER_MODEL = "users.User"
+AUTH_TOKEN_TTL = 24  # Hours
```

Comment:
We should figure out what the default should be. Also, this would be great to have as an environment variable, but I want to wait to deal with that until we remove/refactor `Environment.py` and use the `django-environ` plugin.

Reply:
Praise: Thanks for doing this.  I believe the default should be 4 hours but I'll get confirmation on that.  It's definitely good to be merged as is for the time being.  Of course, validating that the expiration is working is more difficult the longer that the token lives.

Reply:
Awesome, let me know. Super easy to change and since we have it as a settings value (which I'm pretty sure makes it configurable for tests) it should be very easy to write a test for it. I just forgot lol. Let me see about adding a test for this. 

I did confirm that it works "manually" since I already had tokens older than that value.

Reply:
All good: https://github.com/CMSgov/blueprint_api/pull/66/commits/8d0ff09d0cd2a4e72e36b57372a51274b0ae0a1f